Model/Resolver/OrderTotal.php

```diff
@@ -0,0 +1,209 @@
+        array $args = null
+    ) {
+        if (!(($value['model'] ?? null) instanceof OrderInterface)) {
+            throw new LocalizedException(__('"model" value should be specified'));
+        }
+
+        /** @var OrderInterface $order */
+        $order = $value['model'];
+        $currency = $order->getOrderCurrencyCode();
+        $baseCurrency = $order->getBaseCurrencyCode();
+
+        return [
+            'base_grand_total' => ['value' => $order->getBaseGrandTotal(), 'currency' => $baseCurrency],
+            'grand_total' => ['value' => $order->getGrandTotal(), 'currency' => $currency],
+            'subtotal' => ['value' => $order->getSubtotal(), 'currency' => $currency],
+            'total_tax' => ['value' => $order->getTaxAmount(), 'currency' => $currency],
+            'taxes' => $this->getAppliedTaxesDetails($order),
+            'discounts' => $this->getDiscountDetails($order),
+            'total_shipping' => ['value' => $order->getShippingAmount(), 'currency' => $currency],
+            'shipping_handling' => [
+                'amount_excluding_tax' => [
+                    'value' => $order->getShippingAmount(),
+                    'currency' => $order->getOrderCurrencyCode()
+                ],
+                'amount_including_tax' => [
+                    'value' => $order->getShippingInclTax(),
+                    'currency' => $currency
+                ],
+                'total_amount' => [
+                    'value' => $order->getShippingAmount(),
+                    'currency' => $currency
+                ],
+                'taxes' => $this->getAppliedShippingTaxesDetails($order),
+                'discounts' => $this->getShippingDiscountDetails($order),
+            ],
+            'model' => $order
+        ];
+    }
+
+    /**
+     * Retrieve applied taxes that apply to the order
+     *
+     * @param OrderInterface $order
+     * @return array
+     */
+    private function getAllAppliedTaxesOnOrders(OrderInterface $order): array
+    {
+        $extensionAttributes = $order->getExtensionAttributes();
+        $appliedTaxes = $extensionAttributes->getAppliedTaxes() ?? [];
+        $allAppliedTaxOnOrders = [];
+        foreach ($appliedTaxes as $taxIndex => $appliedTaxesData) {
+            $allAppliedTaxOnOrders[$taxIndex] = [
+                'title' => $appliedTaxesData->getDataByKey('title'),
+                'percent' => $appliedTaxesData->getDataByKey('percent'),
+                'amount' => $appliedTaxesData->getDataByKey('amount'),
+            ];
+        }
+        return $allAppliedTaxOnOrders;
+    }
+
+    /**
+     * Return taxes applied to the current order
+     *
+     * @param OrderInterface $order
+     * @return array
+     */
+    private function getAppliedTaxesDetails(OrderInterface $order): array
+    {
+        $allAppliedTaxOnOrders = $this->getAllAppliedTaxesOnOrders($order);
+        $taxes = [];
+        foreach ($allAppliedTaxOnOrders as $appliedTaxes) {
```

Comment:
add spaces after and before foreach, if, while and etc blocks if there is other operations.

Reply:
newlines before foreach/if/return & so on statements are also appreciated